site/docs/v1/tech/lambdas/epic-games-reconcile.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: doc
+title: Epic Games Reconcile
+description: An overview of the Epic Games Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Epic Games Reconcile lambda
+
+When an Epic Games identity provider is used to complete a federated login request FusionAuth will use the email address to reconcile user. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
+// TODO: Confirm that it is email 👆
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement.
+
+If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function reconcile(user, registration, idToken) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
+
+* `user` - the FusionAuth User object
+* `registration` - the FusionAuth UserRegistration object
+* `idToken` - the JSON payload returned by the Epic Token Info API
```

Comment:
This should be `accessToken`, the lambda type enum is wrong. 

https://github.com/FusionAuth/fusionauth-api/blob/8ebf1ab672fb56a16285ce97dddc5c34e2b525fc/src/main/java/io/fusionauth/api/service/authentication/EpicGamesIdentityProviderAuthenticationService.java#L79
https://github.com/FusionAuth/fusionauth-java-client/blob/39ed46a3380816695d987988f676dc37df51abfe/src/main/java/io/fusionauth/domain/LambdaType.java#L232